mmv1/templates/terraform/examples/bigtable_app_profile_multicluster.tf.erb

```diff
@@ -19,6 +19,11 @@ resource "google_bigtable_instance" "instance" {
     storage_type = "HDD"
   }
 
+  timeouts {
```

Comment:
It seems a little strange to include this in the app_profile example, if you're trying to verify that these fields are settable in a test. I would recommend either dropping this change (we rarely test setting timeouts, as they are built-in Terraform primitives), or add this somewhere in `mmv1/third_party/terraform/tests/resource_bigtable_instance_test.go`.

Reply:
Removed :)